scripts/longest_pole.py

```diff
@@ -0,0 +1,69 @@
+"""
+This Python script computes and prints the "Longest Pole" as described in
+[this thread](https://leanprover.zulipchat.com/#narrow/stream/113488-general/topic/The.20long.20pole.20in.20mathlib/near/253932389).
+
+It assumes the presence of a file "scripts/times.json" containing data as
+found, for example, [here](https://mathlib-bench.limperg.de/commit/a5d2dbc51bbf61b715b09552717001c6c5ae653b/json)
+
+This implementation depends on the NetworkX python module.
+"""
+
+import json
+import os
+import networkx as nx
+
+timings = json.load(open("scripts/times.json"))["timings"]
+
+import_graph = nx.DiGraph()
+
+timing_filename_set = set(timings)
+
+for filename in timings:
+    import_graph.add_node(filename, weight=timings[filename])
+
+print(f"{len(import_graph.nodes)} timings added")
+
+count = 0
+
+walked_set = set()
+
+for root, directories, filenames in os.walk('src'):
+
+    for filename in filenames:
+
+        if filename.endswith(".lean"):
```

Comment:
Likely easier as
```suggestion
for filename in Path('src').glob('*.lean'):
```
with `from pathlib import Path` at the top of the file.

Reply:
done